CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `+=`, `-=`, `*=`, `/=` to add, subtract, multiply and divide measurements. [#1162](https://github.com/SwifterSwift/SwifterSwift/pull/1162) by [Roman Podymov](https://github.com/RomanPodymov)
 - **UIView**
   - Added `removeBlur()` method for removing the applied blur effect from the view. [#1159](https://github.com/SwifterSwift/SwifterSwift/pull/1159) by [regi93](https://github.com/regi93)
+  - Added `makeCircle(diameter:)` method to make the view circular. [#1165](https://github.com/SwifterSwift/SwifterSwift/pull/1165) by [happyduck-git](https://github.com/happyduck-git)
```

Comment:
Why are these included?

Reply:
I think this and the 3rd change request was due to my fork being synced before #1165 was merged. I'll look into it soon.